app.js

```diff
@@ -25,6 +27,15 @@ app.use(cors());
 app.use('/', indexRouter);
 app.use('/users', usersRouter);
 app.use('/cars', carsRouter);
+app.use('/saleposts', salePostsRouter);
+
+app.get("/images/:key", (req, res) => {
```

Comment:
it is better to separate this images from app.js. keep them in the router folder.